shipyard_blueprints/fivetran/shipyard_fivetran/cli/update_connector.py

```diff
@@ -61,7 +63,7 @@ def main():
             **args_dict,
         )
     except ExitCodeException as e:
-        fivetran_client.logger.error(e)
```

Comment:
Instead of just logging the exception, put a message in there like we do for the others (i.e. "Something went wrong when attempting to do xyz, message from the server/api: {e}")

Reply:
Tried making minimal code change in order to introduce breaking changes.

There is already a line that logs
`logger.error(f"Error: {resp.status_code} - {resp.text}")` 

and the exception that should get logged when an unknown error occurs is 
`raise ExitCodeException(f"{resp.json()}", self.EXIT_CODE_UNKNOWN_ERROR)`

The others use `.get` which in theory could return None so I think it is best to not use the typical "Message from server"

but I did wrap the error message with some english to hopefully make it easier for the end user to follow